tools/triage/add_triage_signature.py

```diff
@@ -1323,6 +1324,78 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket(report)
 
 
+class IpChangedAfterReboot(ErrorSignature):
```

Comment:
How is this signature different from [this](https://github.com/openshift/assisted-installer/blob/61e5f341f575a01c3fe6f8851224a389b62df261/src/common/common.go#L175) existing warning?

Reply:
This works after node joins.  The signature detects also cases if node didn't join.